packages/worker/src/api/controllers/global/tenant.ts

```diff
@@ -8,3 +8,12 @@ export const save = async (ctx: Ctx<TenantInfo>) => {
     _rev: response.rev,
   }
 }
+
+export const get = async (ctx: Ctx) => {
+  const response = await tenancy.getTenantInfo(ctx.params.id)
+  if (response) {
+    ctx.body = response
+  } else {
+    ctx.throw(404, "Tenant info not found.")
```

Comment:
Does this do what we think it does? If tenant info call throws (404, 500 etc) I believe it will skip this block altogether and throw the error up the chain. Worth testing it